docs/conf.py

```diff
@@ -126,11 +137,61 @@
 # If true, keep warnings as "system message" paragraphs in the built documents.
 # keep_warnings = False
 
+##########################################################################
+## Extension Configuration
+##########################################################################
+
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = False
 
+# Auto-plot settings either as extension or (file format, dpi)
+plot_formats = [
+    'png',
+    'pdf',
+    # ('hires.png', 350),
+]
+
+# By default, include the source code generating plots in documentation
+plot_include_source = True
+
+# Whether to show a link to the source in HTML.
+plot_html_show_source_link = True
+
+# Code that should be executed before each plot.
+plot_pre_code = (
+    "import numpy as np\n"
+    "import matplotlib.pyplot as plt\n"
+    "from yellowbrick.datasets import *\n"
+)
+
+# Whether to show links to the files in HTML.
+plot_html_show_formats = True
 
-# -- Options for HTML output ----------------------------------------------
+# A dictionary containing any non-standard rcParams that should be applied before each plot.
+plot_rcparams = {
+    "figure.figsize": (9,6),
+    "figure.dpi": 128,
+}
```

Comment:
This solved the figure size problem in the docs.

Reply:
good question - this is just for the documentation and I set the figure width to be about the same size as the body of the web page. I think that these are just defaults and can be configured in the `.. plot::` directive, e.g. with 

```python
_, ax = plt.subplots(figsize=(12,4))
```

But we should test that out!

Reply:
It looks like you can also specify plot sizes in the directive as options.

From docs:

"Additionally, this directive supports all of the options of the image directive, except for target (since plot will add its own target). These include alt, height, width, scale, align and class."

Reply:
That will be very helpful for the gallery plots!